pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -901,6 +901,20 @@ impl CustomScan for PdbScan {
         // Remove the oid from the with_index object
         // This helps to reduce the variability of the explain output used in regression tests
         Self::cleanup_varibilities_from_tantivy_query(&mut json_value);
+
+        // Add a flag to indicate if the query is a full index scan
+        if state
+            .custom_state()
+            .base_search_query_input()
+            .is_full_scan_query()
+        {
+            json_value.as_object_mut().unwrap().shift_insert(
```

Comment:
This implementation doesn't seem to line up with the regress output, which is strange?

Reply:
Hm. Yea, for some reason the tests haven't run.